composer.lock

```diff
@@ -1089,34 +1089,37 @@
         },
         {
             "name": "ocramius/package-versions",
-            "version": "2.1.0",
+            "version": "1.9.0",
```

Comment:
This change is wrong, @dependabot.

Reply:
Yeap... this is happening quite often, honestly.
It seems they aren't using the min supported PHP versions to run composer (`7.4`). Then, everything gets messed up.